nabweb/templates/nabweb/help.html

```diff
@@ -9,17 +9,19 @@ <h1 class="display-5">{% trans "Nabaztag help" %}</h1>
       <p><b>{% trans "Base LED pulse color" %}</b> {% trans "indicates the rabbit's network connection state" %}<br>{% blocktrans %}<span class="badge badge-pill badge-info">Fuchsia</span> means <i>all is fine</i>: rabbit has an IP address on your local network, and access to the Internet.<br><span class="badge badge-pill badge-warning">Orange</span> means <i>not so good</i>: rabbit has a local IP address, but no Internet connection (check your router).<br><span class="badge badge-pill badge-danger">Red</span> means <i>real bad</i>: rabbit has no IP address (check your WiFi access point / DHCP server).{% endblocktrans %}</p>
       <hr class="my-4">
       <p class="lead">{% trans "On the Internet" %}</p>
       <p><b>{% trans "<a target='_blank' href='https://tinyletter.com/nabaztag' rel='noopener noreferrer'>Nabaztag is back!</a>" %}</b><br>{% trans "Subscribe to the newsletter." %}</p>
+      <p><b>{% trans "<a target='_blank' href='https://dsc.gg/nabaztagtagtag' rel='noopener noreferrer'>Discord Nabaztag & Tag:Tag:Tag</a>" %}</b><br>{% trans "Modern place to discuss." %}</p>
       <p><b>{% trans "The <a target='_blank' href='https://www.tagtagtag.fr/forum/' rel='noopener noreferrer'>Tag:Tag:Tag user forum</a>" %}</b><br>{% trans "Share your problems and solutions with other rabbit breeders." %}</p>
       <p><b>{% trans "The <a target='_blank' href='https://github.com/nabaztag2018/pynab/#readme' rel='noopener noreferrer'>GitHub repository</a>" %}</b><br>{% trans "Report bugs or contribute to the project with other developers." %}</p>
       <p><b>{% trans "<a target='_blank' href='https://twitter.com/nabaztagtagtag' rel='noopener noreferrer'>Nabaztag on Twitter</a>" %}</b></p>
       <p><b>{% trans "The <a target='_blank' href='https://www.instructables.com/member/tagtagtag/' rel='noopener noreferrer'>TagTagTag assembly instructions</a>" %}</b><br>{% trans "The open heart operation to transplant the card into your rabbit. At this stage you probably do not need this anymore..." %}</p>
-      <p><b>{% trans "The <a target='_blank' href='https://www.tagtagtag.fr/index_eng.html' rel='noopener noreferrer'>Eternal Sunshine of the Nabaztag</a>" %}</b><br>{% trans "The TagtagTag project..." %}</p>
+      <p><b>{% trans "<a target='_blank' href='https://fr.ulule.com/le-retour-du-retour-du-nabaztag/' rel='noopener noreferrer'>Nabaztag is BackBack</a>" %}</b><br>{% trans "Some project history..." %}</p>
+      <p><b>{% trans "<a target='_blank' href='https://fr.ulule.com/l-eternel-retour-du-nabaztag/' rel='noopener noreferrer'>The eternal return of the Nabaztag</a>" %}</b><br>{% trans "Collection 2021-2022" %}</p>
```

Comment:
I do not agree with this change, which basically reverts the changes that were made in PR #298 [remove direct references to Ulule campaigns (which come and go...) and replace them by a reference to the tagtagtag.fr project page (which itself references the Ulule campaigns)].